src/targets/gpu/target.cpp

```diff
@@ -128,6 +130,8 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         auto_contiguous{},
         optimize_module{},
         enable_pass(not enabled(MIGRAPHX_DISABLE_POINTWISE_FUSION{}), fuse_pointwise{}),
+        enable_pass(not enabled(MIGRAPHX_DISABLE_REDUCE_FUSION{}), fuse_reduce{}),
+        dead_code_elimination{},
         dead_code_elimination{},
```

Comment:
Why do we need two dead code eliminations? Shouldint there be a dead_code_elimination after the fuse_pointwise_pass?